frame/staking/src/benchmarking.rs

```diff
@@ -134,25 +134,25 @@ benchmarks! {
 	bond_extra {
 		let (stash, controller) = create_stash_controller::<T>(USER_SEED, 100, Default::default())?;
 		let max_additional = T::Currency::minimum_balance() * 10u32.into();
-		let ledger = Ledger::<T>::get(&controller).ok_or("ledger not created before")?;
```

Comment:
None of the changes in this file makes any sense. This is a static string and nothing will be allocated here.

Reply:
Fixed.